endpoints_proto_datastore/utils.py

```diff
@@ -428,8 +428,8 @@ class _EPDProtoJson(protojson.EndpointsProtoJson):
   are stored on the newly created message.
   """
 
-  def decode_message(self, message_type, encoded_message):
-    """Merge JSON structure to Message instance.
+  def _ProtoJson__decode_dictionary(self, message_type, msg_dictionary):
```

Comment:
I'm puzzled why this is _ProtoJson__decode_dictionary rather than just decode_dictionary...?